tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,977 @@
+<tool id="astronomical_archives" name="Astronomical Archives (IVOA)" version="0.9.0">
+    <description>queries astronomical archives through Virtual Observatory protocols</description>
+    <edam_operations>
+        <edam_operation>operation_0224</edam_operation>
+    </edam_operations>
+    <requirements>
+        <requirement type="package" version="5.2.2">astropy</requirement>
+        <requirement type="package" version="1.4.1">pyvo</requirement>
+    </requirements>
+    <command>
+      <![CDATA[
+
+        mkdir fits
+
+        &&
+
+        python '$__tool_directory__/astronomical_archives.py' '$output' '$output_csv' '$output_html' '$output_basic_html' '$output_error' inputs.json
+
+      ]]>
+    </command>
+    <configfiles>
+        <inputs name="inputs" filename="inputs.json" />
+    </configfiles>
+    <inputs>
+        <conditional name="archive_selection">
+            <param name="archive_type" type="select" label="Archive Selection">
+              <option value="archive">Query specific IVOA archive</option>
+              <option value="registry">Query IVOA registry</option>
+            </param>
+            <when value="registry">
+              <param name="keyword" type="text" label="Keyword" />
+              <param name="service_type" type="select" label="Service type">
+                <option value="TAP" selected="true">TAP: Tables</option>
+                <option value="SCS">SCS: Cone search</option>
+                <option value="SSA">SSA: Spectra</option>
+                <option value="SIA">SIA: Images</option>
+              </param>
+              <param name="wavebands" type="select" label="Wavebands">
+                <option value="all" selected="true">All</option>
+                <option value="radio">Radio</option>
+                <option value="millimeter">Millimeter</option>
+                <option value="infrared">Infrared</option>
+                <option value="optical">Optical</option>
+                <option value="uv">UV</option>
+                <option value="euv">EUV</option>
+                <option value="x-ray">X-ray</option>
+                <option value="gamma-ray">Gamma-ray</option>
+              </param>
+            </when>
+            <when value="archive">
+              <param name="archive" type="select" label="Astronomical archives">
+                <options from_data_table="astronomical_archives_1" />
```

Comment:
why has this a _1 ?

Reply:
my bad thanks, forgot to change after moving the code